include/NovelRT/Ecs/Audio/AudioEmitterStateComponent.h

```diff
@@ -0,0 +1,25 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_ECS_AUDIO_AUDIOEMITTERSTATECOMPONENT_H
+#define NOVELRT_ECS_AUDIO_AUDIOEMITTERSTATECOMPONENT_H
+
```

Comment:
Are we ignoring the namespace error guard for this namespace since its legacy?

Reply:
No, this should have a guard akin to the ECS ones even if it is legacy. i'll fix it